pkgs/applications/version-management/guilt/default.nix

```diff
@@ -0,0 +1,94 @@
+{ asciidoc
+, docbook_xml_dtd_45
+, docbook_xsl
+, fetchFromGitHub
+, gawk
+, git
+, gnused
+, lib
+, makeWrapper
+, perl
+, stdenv
+, xmlto
+}:
+
+let
+  version = "0.37-rc1";
+  sha256 = "sha256-7OgRbMGYWtGvrZxKfJe0CkpmU3AUkPebF5NyTsfXeGA=";
+in
+stdenv.mkDerivation rec {
+  pname = "guilt";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "jeffpc";
+    repo = "guilt";
+    rev = "v${version}";
+    inherit sha256;
+  };
+
+  doCheck = true;
```

Comment:
Is this required? Isn't it true by default?

Reply:
[It is not](https://nixos.org/manual/nixpkgs/stable/#var-stdenv-doCheck).

Reply:
Ah. TIL! Thanks!